members/CTL000037.yaml

```diff
@@ -36,8 +40,10 @@ contact_form:
     - click_on:
         - value: Submit
           selector: form#webform-client-form-1797 div.form-actions input.webform-submit
+    - wait:
+        - value: 7
   success:
     headers:
       status: 200
     body:
-      contains: Thank you for your message.
+      contains: Go back to the form
```

Comment:
does this message ONLY appear if the form submits successfully?

Reply:
I believe so. It's in the thank you page supporters get once the msg is submitted (see image below). I don't see the 'Go back to form' message anywhere in the page. 

![image](https://user-images.githubusercontent.com/8680733/39638476-4dca3300-4f94-11e8-823e-81ec2fde5e23.png)

Reply:
I see "Thank your for your message." there. Why did you change it?